bindings/Sofa/src/SofaPython3/Sofa/Core/Binding_Node.cpp

```diff
@@ -32,6 +32,7 @@ using sofa::core::objectmodel::BaseObjectDescription;
 #include <sofa/core/objectmodel/Link.h>
 
 #include "Binding_Node_doc.h"
+#include <SofaPython3/Sofa/Core/PythonScriptEvent.h>
```

Comment:
Not a big issue here, but when including a header file belonging to the same target / library, as it's the case here, you should include using `""` and not `<>` (`#include "PythonScriptEvent.h"`)